lib/sidekiq/dynamic_queues/attributes.rb

```diff
@@ -109,7 +122,24 @@ def expand_queues(queues)
           end
         end
 
-        return matched_queues.collect { |q| "queue:#{q}" }.uniq
+        return matched_queues.sort
+      end
+
+      def expand_queues_for_display(queues)
```

Comment:
It looks like this method is trying to get a unique list of queues since they're created multiple times using the same name.  Is there are reason you're not using `.uniq`?